kapitan/inputs/binary.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 The Kapitan Authors
+# SPDX-FileCopyrightText: 2020 The Kapitan Authors <kapitan-admins@googlegroups.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import subprocess
+import re
+import logging
+import os
+import shutil
+from distutils.dir_util import copy_tree
+
+from kapitan.inputs.base import InputType
+
+logger = logging.getLogger(__name__)
+
+
+class Binary(InputType):
```

Comment:
I know this is a bit late. But what do you think of calling this "External" instead? I think it's more accurate than "binary", since it runs an external executable, which could be a script or a binary.